serverSrc/src/templates/app.handlebars

```diff
@@ -6,6 +6,10 @@
     <link rel="shortcut icon" href="assets/icons/favicon-32x32.png" type="image/x-icon"/>
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
     <title>{{appTitle}}</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"/>
```

Comment:
Have a look at react bootstrap ;)

Reply:
are we using bootstrap in this project? Do we have option to use what we like? 
In real life do we have option or we must use what everybody use?

Reply:
> Have a look at react bootstrap ;)

are we using bootstrap in this project? Do we have option to use what we like?
In real life do we have option or we must use what everybody use?

Reply:
You guys can decide, I mentioned react bootstrap because I saw that bootstrap was imported in the line above.